### PR TITLE
OSDOCS#12377: Release notes for OSSO 1.1.4

### PR DESCRIPTION
**Pull request by bergerhoffer**
Version(s):
4.12, 4.13

Issue:
https://issues.redhat.com/browse/OSDOCS-12377

Link to docs preview:
https://85239--ocpdocs-pr.netlify.app/openshift-enterprise/latest/nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.html#secondary-scheduler-operator-release-notes-1.1.4

QE review:
- [x] QE has approved this change.

Additional information:

**Advisory link won't work until the release date, currently scheduled for november 26**

**Known issue is copy/paste from earlier releases**

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@kasturinarra @ingvagabund PTAL at OSSO 1.1.4, thanks!

https://85239--ocpdocs-pr.netlify.app/openshift-enterprise/latest/nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.html#secondary-scheduler-operator-release-notes-1.1.4

**Comment by ingvagabund** [comment]
LGTM

File: nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.adoc [hunk 1]
```diff
@@ -12,6 +12,33 @@ These release notes track the development of the {secondary-scheduler-operator-f
 
 For more information, see xref:../../../nodes/scheduling/secondary_scheduler/index.adoc#nodes-secondary-scheduler-about_nodes-secondary-scheduler-about[About the {secondary-scheduler-operator}].
 
+[id="secondary-scheduler-operator-release-notes-1.1.4"]
+== Release notes for {secondary-scheduler-operator-full} 1.1.4
+
+Issued: 25 November 2024
```
Comment (kasturinarra) [inline nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.adoc:18]: @bergerhoffer just one comment, could you  please help change the date as this is changed based on my request yesterday ?, thank you !!

Comment (bergerhoffer) [reply nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.adoc]: Updated, thanks! (Will take a few mins for the preview to update)

**Comment by michaelryanpeter** [comment]
LGTM
/label peer-review-done
/remove-label peer-review-in-progress
/remove-label peer-review-needed

**Pull request merged**